aws/ansible/templates/ote/config.edn.j2

```diff
@@ -11,4 +11,6 @@
            :name-field "NAMEFIN"
            :url "https://s3.eu-central-1.amazonaws.com/ote-assets/places/finnish-municipalities/SuomenKuntajako_2017_10k.shp"}}
  :nap {:api "{{base_url}}/api/"
-       :export-base-url "{{base_url}}"}}
+       :export-base-url "{{base_url}}"}
+ :log {:level :info}
```

Comment:
could we use warning level in production?

Reply:
we could but should we?

Reply:
I think that we only need production logs when we have problems and warnings and errors are enough in those situations.
So I would suggest that we log only things we need and not everything